core/finality-grandpa/Cargo.toml

```diff
@@ -22,7 +22,7 @@ network = { package = "substrate-network", path = "../network" }
 service = { package = "substrate-service", path = "../service", optional = true }
 srml-finality-tracker = { path = "../../srml/finality-tracker" }
 fg_primitives = { package = "substrate-finality-grandpa-primitives", path = "primitives" }
-grandpa = { package = "finality-grandpa", version = "0.6.0", features = ["derive-codec"] }
+grandpa = { package = "finality-grandpa", path = "../../../finality-grandpa", features = ["derive-codec"] }
```

Comment:
Is this intentional? Seems like you might have forgotten to switch back to the version after developing locally.

Reply:
It was waiting on `0.7.0` to be published in fact, which it should have been now.